code/defines/procs/announcement.dm

```diff
@@ -119,17 +124,28 @@
 
 	if(!isnull(signature))
 		message += "<br><br><i> Signed by, <br> [signature]</i>"
+	else
```

Comment:
I don't think the else and indention here is correct unless you want a signed message to always go through (only place I see currently that can have the signature as something is in /obj/structure/machinery/computer/almayer_control/Topic )

Reply:
I did this over two months ago, but as I recall it was due to some duplication error. I'll debug it again later.